src/ngraph/runtime/gpu/gpu_cuda_kernel_builder.cpp

```diff
@@ -23,9 +23,9 @@ using namespace ngraph;
 void runtime::gpu::CudaKernelBuilder::get_elementwise_op(codegen::CodeWriter& writer,
                                                          const std::string& name,
                                                          const std::string& op,
-                                                         const std::vector<std::string>& data_types,
-                                                         const size_t& num_inputs)
+                                                         const std::vector<std::string>& data_types)
 {
+    auto num_inputs = data_types.size() - 1;
```

Comment:
need to double check if output size will always be one.

Reply:
This cannot handle multi-output ew ops. Is there an example of one that we need to support? I can add an assertion just in case.

Reply:
Added the check in GPU_Emitter (https://github.com/NervanaSystems/ngraph/pull/979/files#diff-88bf3ba31e0b08d55a3811bc6932533aR72)